koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -269,96 +269,12 @@ def format_label(self, label_id):
-                "description": "Build target."
-            },
-            {
-                "type": ["object", "null"],
-                "properties": {
-                    "scratch": {
-                        "type": "boolean",
-                        "description": "Perform a scratch build?"
-                    },
-                    "isolated": {
-                        "type": "boolean",
-                        "description": "Perform an isolated build?"
-                    },
-                    "yum_repourls": {
-                        "type": ["array", "null"],
-                        "items": {
-                            "type": "string"
-                        },
-                        "description": "URLs of yum repo files."
-                    },
-                    "git_branch": {
-                        "type": ["string", "null"],
-                        "description": "Git branch to build from."
-                    },
-                    "push_url": {
-                        "type": ["string", "null"]
-                    },
-                    "koji_parent_build": {
-                        "type": ["string", "null"],
-                        "description": "Overwrite parent image with image from koji build."
-                    },
-                    "release": {
-                        "type": ["string", "null"],
-                        "description": "Set release value."
-                    },
-                    "flatpak": {
-                        "type": "boolean",
-                        "description": "Build a flatpak instead of a container?"
-                    },
-                    "compose_ids": {
-                        "type": ["array", "null"],
-                        "items": {
-                            "type": "integer"
-                        },
-                        "description": "ODCS composes used."
-                    },
-                    "signing_intent": {
-                        "type": ["string", "null"],
-                        "description": "Signing intent of the ODCS composes."
-                    },
-                    "skip_build": {
-                        "type": "boolean",
-                        "description": "Skip build, just update buildconfig for autorebuild "
-                                       "and don't start build"
-                    },
-                    "triggered_after_koji_task": {
-                        "type": "integer",
-                        "description": "Koji task for which autorebuild runs"
-                    }
-                }
-            }
-        ]
-    }
-
-    def __init__(self, id, method, params, session, options, workdir=None, demux=True):
-        BaseTaskHandler.__init__(self, id, method, params, session, options,
-                                 workdir)
+class BaseContainerTask(BaseTaskHandler):
+    # Common class for BuildContainerTask and BuildSourceContainerTask
```

Comment:
Could this  (and all other comments after class definitions) become an actual docstring?

Reply:
sure